public/index.php

```diff
@@ -112,12 +188,23 @@ class_alias('Fuel\\Core\\Autoloader', 'Autoloader');
 	$response = $routerequest('_500_', $e);
 }
 
-// This will add the execution time and memory usage to the output.
-// Comment this out if you don't use it.
+/**
+ * -----------------------------------------------------------------------------
+ *  Start profiling
+ * -----------------------------------------------------------------------------
+ *
+ *  This will add the execution time and memory usage to the output.
+ *
+ *  Comment these out if you don't use it.
+ *
+ */
+
 $response->body((string) $response);
```

Comment:
This line should be outside this docblock, if you comment this line, you will never get any output.

Reply:
I have moved the code from profiling logic block in the [second commit](
https://github.com/fuel/fuel/pull/319/commits/5968aab6373938882d41a3b64d4309ac64ad813a). Please check whether I have done it correctly